requirements_all.txt

```diff
@@ -933,6 +923,9 @@ pyzabbix==0.7.4
 # homeassistant.components.sensor.qnap
 qnapstats==0.2.4
 
+# homeassistant.components.sensor.qq
+qqbot==2.3.7
```

Comment:
Keep this. The rest of the file needs to be reverted

Reply:
Is this ok?

Reply:
You see,I recover it，just now

Reply:
No, there are still a lot of changes in this file that needs to be reverted.